tock/tock/templates/_navigation.html

```diff
@@ -1,51 +1,54 @@
-<nav role="navigation" class="usa-nav">
+<nav class="usa-nav" aria-label="Primary navigation">
   <div class="usa-nav__inner">
     <button class="usa-nav__close">
       {% load static %}
       <img src="{% static 'vendor/uswds_v2.3.1/img/close.svg' %}" alt="close">
     </button>
     <ul class="usa-nav__primary usa-accordion">
       <li class="usa-nav__primary-item">
-        <a href="{% url 'ProjectListView' %}" class="usa-nav__link">
-          <span>Projects</span>
+        <a href="/" accesskey="1" title="Home" aria-label="Home">Tock your time</a>
+      </li>
+      <li class="usa-nav__primary-item">
+        <a class="usa-nav__link" href="{{ x_tock_change_request_form_url }}" target="_blank">
+          <span class="usa-link--external">Request changes to your time</span>
         </a>
       </li>
       {% if request.user.is_staff %}
       <li class="usa-nav__primary-item">
         <button class="usa-accordion__button usa-nav__link" aria-expanded="false" aria-controls="reports-nav">
-          <span>Reports</span>
+          <span>Review summaries</span>
         </button>
         <ul id="reports-nav" class="usa-nav__submenu" aria-expanded="true">
           <li class="usa-nav__submenu-item">
             <a href="{% url 'utilization:GroupUtilizationView' %}">Utilization reports</a>
           </li>
           <li class="usa-nav__submenu-item">
             <a href="{% url 'reports:ListReports' %}" class="usa-nav__link">
-              <span>Timecard Reports</span>
+              <span>Timecard reports</span>
```

Comment:
Thank you

Reply:
![of course](https://media.giphy.com/media/pZo6MivdRcoeI/giphy.gif)